specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/securitydomain.json

```diff
@@ -0,0 +1,244 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "KeyVaultClient",
+        "description": "The key vault client performs cryptographic key operations and vault operations against the Key Vault service.",
+        "version": "7.2-preview"
+    },
+    "x-ms-parameterized-host": {
+        "hostTemplate": "{vaultBaseUrl}",
+        "useSchemePrefix": false,
+        "positionInOperation": "first",
+        "parameters": [
+            {
+                "name": "vaultBaseUrl",
+                "description": "The vault name, for example https://myvault.vault.azure.net.",
+                "required": true,
+                "type": "string",
+                "in": "path",
+                "x-ms-skip-url-encoding": true
+            }
+        ]
+    },
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/securitydomain/download": {
+            "post": {
+                "tags": [
+                    "HSMSecurityDomain"
+                ],
+                "operationId": "HSMSecurityDomainDownload",
+                "description": "Retrieves Security domain from HSM enclave",
+                "parameters": [
+                    {
+                        "in": "body",
+                        "name": "Certificates",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/CertificateSet"
+                        },
+                        "description": "Security domain download operation requires customer to provide three certificates in PEM format"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Security domain backup response",
```

Comment:
Stay consistent with terminology. Call this Security domain 'download' response.